components/favicon_title_handler/favicon_title_handler.tsx

```diff
@@ -179,6 +185,22 @@ export class FaviconTitleHandlerClass extends React.PureComponent<Props> {
         }
     }
 
+    updateAppBadge = (unreadStatus: BasicUnreadStatus) => {
+        if (!navigator.setAppBadge) {
```

Comment:
```suggestion
        if (UserAgent.isDesktopApp() || !navigator.setAppBadge) {
```

Still not sure if this is needed. But this would be a way to address the possible Electron issue.